NSS-Therapeutic-Raccoons/WorkforceManagement
# Jp employee detail

## Link to Ticket
Closes  #4 

## Description of Proposed Changes
- Created mvc for employee details view
- Created integration test for Employee details View

## Steps to Test

Outline the steps to test

```sh
git fetch --all
git checkout JPEmployeeDetail
run the test for Get_DetailContentVerified
verify in browser that all fields are generated for specific User.
On webpage click on employees, then details(for a specific employee)
```


## Impacted Areas in Application

List general components of the application that this PR will affect:

*New MVC and Test. Will not affect other MVCs or Models or Controllers.

## Mentions @username

@khardt
@figamus
@jeremiah3643
@ricky-bruner
@askingalot

## Definition of Done

1. The project must be fully documented. This includes the following:
    1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
    1. Every class must be documented with purpose, author, and methods.
    1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each.
